### PR TITLE
O3-1934: Set up Basic Tests and Test Scaffolding

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: src/components/medication-dispense-review.component.tsx [hunk 1]
```diff
@@ -404,7 +404,6 @@ const MedicationDispenseReview: React.FC<MedicationDispenseReviewProps> = ({
           disabled={!userCanModify}
           hideSteppers={true}
           id="dosingQuanity"
-          light={isTablet}
```
Comment (mogoodrich) [inline src/components/medication-dispense-review.component.tsx:407]: I got a warning that this was no longer supported on this element (I think a NumberInput?) so I just removed it.

File: src/medication-request/medication-request.resource.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ import {
   getMedicationDisplay,
   getMedicationReferenceOrCodeableConcept,
   getPrescriptionTableActiveMedicationRequestsEndpoint,
-  getPrescriptionTableAllMedicationRequestsEnpointEndpoint,
+  getPrescriptionTableAllMedicationRequestsEndpoint,
```
Comment (mogoodrich) [inline src/medication-request/medication-request.resource.tsx:17]: fixed a typo

File: src/medication-request/medication-request.resource.tsx [hunk 3]
```diff
@@ -140,7 +140,7 @@ function buildPrescriptionsTableRow(
   };
 }
 
-function computePrescriptionStatus(
+export function computePrescriptionStatus(
```
Comment (mogoodrich) [inline src/medication-request/medication-request.resource.tsx:143]: I exported this so I could test it separately.  Not sure if that's best practice or not, but seemed low risk.

File: src/types.ts [hunk 4]
```diff
@@ -98,45 +98,35 @@ export interface Attribute {
 
 export interface CodeableConcept {
```
Comment (mogoodrich) [inline src/types.ts:99]: In the test classes I used various actual JSON result sets pulled back from my local dev server and some didn't match the typescript definitions, so I fixed so they were consistent... mainly there were main fields marked in this file as required/mandatory that weren't actually mandatory and not present on the test data.

File: src/medication-request/medication-request.resource.tsx (hunk 3 above)
Comment (ibacher) [reply src/medication-request/medication-request.resource.tsx:143]: It only really becomes an issue if someone imports it somewhere other than in a test...

**Review by ibacher — APPROVED** [review]
This LGTM (though I tend to prefer `it` to `test` and slightly shorter descriptions).

**Review by pirupius — APPROVED** [review]

**Pull request merged**